sdk/eventhub/azure-eventhub-checkpointstoreblob/samples/event_processor_blob_storage_example_with_storage_api_version.py

```diff
@@ -1,3 +1,17 @@
+#!/usr/bin/env python
+
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+"""
+Some environments have different versions of Azure Storage Service API. For instance, Azure Stack Platform
+version 2002 uses Azure Storage Service API 2017-11-07.
+
+Specify api_version when you create the BlobCheckpointStore as shown in this example.
+"""
```

Comment:
I like your accurate and terse comment. Please add "For more information on the Azure Storage service versions supported on Azure Stack Hub, please refer to docs.microsoft.com/en-us/azure-stack/user/azure-stack-acs-differences." Now, I have a version of the same that is a bit more verbose, but provides more context. Please consider using:
"The following code/sample can be used if the environment you are targeting supports a different version of Storage Blob SDK than those typically available on Azure. For example, if you are running Event Hubs on an Azure Stack Hub version 2002, the highest available version for the Storage service is version 2017-11-09. In this case, you will need to use the following code to change the Storage service API version to 2017-11-09. For more information on the Azure Storage service versions supported on Azure Stack Hub, please refer to docs.microsoft.com/en-us/azure-stack/user/azure-stack-acs-differences. "

Reply:
Updated to this text